### PR TITLE
Fix: issue #573 is now fixed, text and icon are not getting clashed. …

### LINKED ISSUE #573: [STORY] UI : The text and icon are getting clashed at the end of the search bar
# Description
image

Here, the text is getting below the icon, the text should only come till before the icon

### PR DESCRIPTION
**Pull request by Ramalhadas**
…Included Clear input button

# Description
Summarize the change.

Fixed #573

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

Tested using  SigLens UI at http://localhost:5122/

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

**Review by nbhavana — COMMENTED** [review]
the changes look good, just one small suggestion. Can you increase the space between the query text and clear button? That will make it very clear that there is a button to clear the query text.
image

**Comment by Ramalhadas** [comment]
> the changes look good, just one small suggestion. Can you increase the space between the query text and clear button? That will make it very clear that there is a button to clear the query text. image

Yes i can do that. Do you also want me to remove the comments on the code?

I don't know if they're necessary.

File: static/css/siglens.css [hunk 1]
```diff
@@ -3169,6 +3169,10 @@ float: right;
 }
 
 .search-db-input,.metrics-input{
+    /* ADDED */
+    padding-left: 1rem;
```
Comment (nbhavana) [inline static/css/siglens.css:3173]: remove the comment

**Review by nbhavana — COMMENTED** [review]
the changes look good, just one small suggestion. Can you increase the space between the query text and clear button? That will make it very clear that there is a button to clear the query text.
image

**Comment by nbhavana** [comment]
> > the changes look good, just one small suggestion. Can you increase the space between the query text and clear button? That will make it very clear that there is a button to clear the query text. image
> 
> Yes i can do that. Do you also want me to remove the comments on the code?
> 
> I don't know if they're necessary.

just remove the comment from the css file

File: static/css/siglens.css [hunk 2]
```diff
@@ -3169,6 +3169,8 @@ float: right;
 }
 
 .search-db-input,.metrics-input{
+    padding-left: 1rem;
+    padding-right: 3.4rem;
     height: 32px;
```
Comment (nbhavana) [inline static/css/siglens.css:3174]: can you increase the padding-right to 4.4rem?

Comment (Ramalhadas) [reply static/css/siglens.css:3174]: yes i can.

**Review by nbhavana — APPROVED** [review]

**Comment by Ramalhadas** [comment]
@nbhavana Thank you for your help and patience.

**Pull request merged**